src/bidiMapper/BidiServer.ts

```diff
@@ -112,31 +127,17 @@ export class BidiServer extends EventEmitter<BidiServerEvents> {
     );
   }
 
-  #processOutgoingMessage = async (messageEntry: OutgoingBidiMessage) => {
```

Comment:
what is the ratio here?

Reply:
Did you mean rationale?

Those are member variables, not methods. Just grouped the member variables closer together - at the top of the class.

Before I did that I tried to convert them to functions (`async #processOutgoingMessages(messageEntry: OutgoingBidiMessage) {...` but that resulted in an error.